static/html/mozorg/lightbeam.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Static HTML page used in an iframe that can install an
+  add-on directly from firstrun pages on mozilla.org
+  (via the aformentioned iframe).
+-->
+<head>
+  <meta charset="utf-8">
+  <title>Lightbeam Add-on Button</title>
+  <style>
+      html, body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #install-link {
+        background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAwAAAAMCAQAAAD8fJRsAAAAPklEQVR4AWP4D4Wv/oMAkITyKZD4Cpb4iiSBA+CWePkfAn+CeEASxmdY+R8CH4LEgSSMTwt/bPk/EQi3wCUAcebY97QtnxsAAAAASUVORK5CYII=) top 12px left 16px no-repeat, linear-gradient(#84C63C, #489615) repeat scroll 0% 0% #489615;
+        border-radius: 6px;
+        box-shadow: 0px 3px rgba(0, 0, 0, 0.1), 0px -4px rgba(0, 0, 0, 0.1) inset;
+        color: #FFF;
+        display: inline-block;
+        font: bold 16px "Trebuchet MS",sans-serif;
+        margin-top: 8px;
+        padding: 8px 16px 12px 38px;
+        text-align: center;
+        text-decoration: none;
+        text-shadow: 0px 1px 0px rgba(0, 0, 0, 0.5);
+      }
+
+      #install-link:active {
+        box-shadow: none;
+        margin-top: 11px;
+      }
+
+      #icon-container {
+        float: left;
+        width: 80px;
+        text-align: center;
+      }
+
+      #content {
+        color: #3C3C3C;
+        font: message-box;
+        font-size: 12px;
+        padding-left: 80px;
+      }
+
+      #content p {
+        display: table-cell;
+        vertical-align: middle;
+        height: 40px;
+      }
+    </style>
+</head>
+<body>
+
+<a href="#" id="install-link">Add to Firefox</a>
+
+<script>
+(function() {
+  'use strict';
+
+  var installLink = document.getElementById('install-link');
+
+  installLink.addEventListener('click', function(e) {
+    e.preventDefault();
+
+    var xpiUrl = 'https://addons.mozilla.org/firefox/downloads/latest/363974/addon-363974-latest.xpi?src=fxfirstrun';
```

Comment:
So this is the url to install lightbeam, but the iconURL` below is the one for lastpass